### PR TITLE
Large width Single Tprime production at LO, T ->tH for Run 2, tempelate cards and script to produce different parameter

### PR DESCRIPTION
**Pull request by kpal99**
Dear @menglu21, all maintainers,

I have added a set of cards and python3 script to make other cards as suggested in [earlier PR](https://github.com/cms-sw/genproductions/pull/3480).
Kindly let me know if any changers are required. 

Message from earlier PR
> Dears,
> I am Kuldeep Pal, working in NISER, India. I along with my supervisor, Dr. Prolay Kumar Mal are working with B2G group. We would like to produce Large width samples for single tprime production at leading order, T further decays to top and Higgs.
> 
> We would like to produce sample using VLQ_v4_4FNS_UFO model for Run 2.
> The mass point requested are: 700, 1000, 1500, 2000 GeV
> and for each mass point four decay width are request: 5, 10, 20, 30% (written as pct in cards)
> and for each set of mass point and decay; three different channels are specified: s-channel (sch), t-channel (tch) and interference between signal and SM backgroud (int).
> 
> We have presented our gen-level study in the B2G group, which can be access [here](https://indico.cern.ch/event/1297817/#10-production-of-large-width-v).
> 
> Kindly let us know if any changes are required. 
> Cheers,

### CODE REVIEW

File: bin/MadGraph5_aMCatNLO/cards/production/13TeV/TprimeBToTH_LH/TprimeBToTH_LH_M1000_decay30pct_int/TprimeBToTH_LH_M1000_decay30pct_int_run_card.dat [hunk 1]
```diff
@@ -0,0 +1,273 @@
...
+     6500     = ebeam2  ! beam 2 total energy in GeV
+#*********************************************************************
+# Beam polarization from -100 (left-handed) to 100 (right-handed)    *
+#*********************************************************************
+        0     = polbeam1 ! beam polarization for beam 1
+        0     = polbeam2 ! beam polarization for beam 2
+#*********************************************************************
+# PDF CHOICE: this automatically fixes also alpha_s and its evol.    *
+#*********************************************************************
+# 'cteq6l1'    = pdlabel     ! PDF set (lhapdf for using lhapdf)
```
Comment (menglu21) [inline bin/MadGraph5_aMCatNLO/cards/production/13TeV/TprimeBToTH_LH/TprimeBToTH_LH_M1000_decay30pct_int/TprimeBToTH_LH_M1000_decay30pct_int_run_card.dat:51]: remove this line

File: bin/MadGraph5_aMCatNLO/cards/production/13TeV/TprimeBToTH_LH/TprimeBToTH_LH_M1000_decay30pct_sch/TprimeBToTH_LH_M1000_decay30pct_sch_run_card.dat [hunk 2]
```diff
@@ -0,0 +1,273 @@
...
+#*********************************************************************
+# Run to generate the grid pack                                      *
+#*********************************************************************
+  .false.     = gridpack  !True = setting up the grid pack
+#*********************************************************************
+# Number of events and rnd seed                                      *
+# Warning: Do not generate more than 1M events in a single run       *
+# If you want to run Pythia, avoid more than 50k events in a run.    *
+#*********************************************************************
+  300000 = nevents ! Number of unweighted events requested
```
Comment (menglu21) [inline bin/MadGraph5_aMCatNLO/cards/production/13TeV/TprimeBToTH_LH/TprimeBToTH_LH_M1000_decay30pct_sch/TprimeBToTH_LH_M1000_decay30pct_sch_run_card.dat:32]: change to 5000

File: bin/MadGraph5_aMCatNLO/cards/production/13TeV/TprimeBToTH_LH/TprimeBToTH_LH_M1000_decay30pct_sch/TprimeBToTH_LH_M1000_decay30pct_sch_run_card.dat [hunk 3]
```diff
@@ -0,0 +1,273 @@
...
+     6500     = ebeam2  ! beam 2 total energy in GeV
+#*********************************************************************
+# Beam polarization from -100 (left-handed) to 100 (right-handed)    *
+#*********************************************************************
+        0     = polbeam1 ! beam polarization for beam 1
+        0     = polbeam2 ! beam polarization for beam 2
+#*********************************************************************
+# PDF CHOICE: this automatically fixes also alpha_s and its evol.    *
+#*********************************************************************
+# 'cteq6l1'    = pdlabel     ! PDF set (lhapdf for using lhapdf)
```
Comment (menglu21) [inline bin/MadGraph5_aMCatNLO/cards/production/13TeV/TprimeBToTH_LH/TprimeBToTH_LH_M1000_decay30pct_sch/TprimeBToTH_LH_M1000_decay30pct_sch_run_card.dat:51]: remove this line

File: bin/MadGraph5_aMCatNLO/cards/production/13TeV/TprimeBToTH_LH/TprimeBToTH_LH_M1000_decay30pct_tch/TprimeBToTH_LH_M1000_decay30pct_tch_run_card.dat [hunk 4]
```diff
@@ -0,0 +1,273 @@
...
+#*********************************************************************
+# Run to generate the grid pack                                      *
+#*********************************************************************
+  .false.     = gridpack  !True = setting up the grid pack
+#*********************************************************************
+# Number of events and rnd seed                                      *
+# Warning: Do not generate more than 1M events in a single run       *
+# If you want to run Pythia, avoid more than 50k events in a run.    *
+#*********************************************************************
+  300000 = nevents ! Number of unweighted events requested
```
Comment (menglu21) [inline bin/MadGraph5_aMCatNLO/cards/production/13TeV/TprimeBToTH_LH/TprimeBToTH_LH_M1000_decay30pct_tch/TprimeBToTH_LH_M1000_decay30pct_tch_run_card.dat:32]: change to 5000

File: bin/MadGraph5_aMCatNLO/cards/production/13TeV/TprimeBToTH_LH/TprimeBToTH_LH_M1000_decay30pct_tch/TprimeBToTH_LH_M1000_decay30pct_tch_run_card.dat [hunk 5]
```diff
@@ -0,0 +1,273 @@
...
+     6500     = ebeam2  ! beam 2 total energy in GeV
+#*********************************************************************
+# Beam polarization from -100 (left-handed) to 100 (right-handed)    *
+#*********************************************************************
+        0     = polbeam1 ! beam polarization for beam 1
+        0     = polbeam2 ! beam polarization for beam 2
+#*********************************************************************
+# PDF CHOICE: this automatically fixes also alpha_s and its evol.    *
+#*********************************************************************
+# 'cteq6l1'    = pdlabel     ! PDF set (lhapdf for using lhapdf)
```
Comment (menglu21) [inline bin/MadGraph5_aMCatNLO/cards/production/13TeV/TprimeBToTH_LH/TprimeBToTH_LH_M1000_decay30pct_tch/TprimeBToTH_LH_M1000_decay30pct_tch_run_card.dat:51]: remove this line

**Comment by kpal99** [comment]
Hi, I have implemented the comments and removed the cards to 2017/13TeV as well.

**Comment by kpal99** [comment]
Hi @menglu21 
I need to produce same sample for Run3 as well. 
I will change the beam energy in run_cards and place the samples in [13p6TeV/single_VLQ/wide_width_NLO](https://github.com/cms-sw/genproductions/tree/master/bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/single_VLQ/wide_width_NLO
)
And I need to a PR in master branch. 
Is this the correct procedure?

**Pull request merged**